Demo/src/main/java/com/braintreepayments/demo/LocalPaymentFragment.java

```diff
@@ -72,16 +72,16 @@ public void launchIdeal(View v) {
         address.setPostalCode("1072 AE");
 
         LocalPaymentRequest request = new LocalPaymentRequest(true);
-        request.setPaymentType("ideal");
-        request.setAmount("1.10");
-        request.setAddress(address);
-        request.setPhone("207215300");
-        request.setEmail("android-test-buyer@paypal.com");
-        request.setGivenName("Test");
-        request.setSurname("Buyer");
+        request.paymentType = "ideal";
+        request.amount = "1.10";
+        request.address = address;
+        request.phone = "207215300";
+        request.email = "android-test-buyer@paypal.com";
+        request.givenName = "Test";
+        request.surname = "Buyer";
         request.setShippingAddressRequired(true);
```

Comment:
Are we planning to keep support for the `.setProperty` syntax for Java merchants or remove it with v5? I think we should have consistency across all modules/properties but not sure what was done for other conversions (cc @sshropshire @tdchow @chinghlinn)

Reply:
Yep, I agree. We should keep the setter functions for java integrations. I think we'll need to move all the properties in the LocalPaymentRequest into the constructor for the setters to be generated.

Reply:
Actually, I think the getter/setter functions aren't generated due to the `@JvmField` annotations on each of the properties.

Reply:
Yeah the `set` prefix is standard for Java. It's probably safe to assume our Java merchants are comfortable with the verbosity if they haven't migrated to Kotlin by now. If we remove the `@JvmField` annotations we can leverage the default Kotlin / Java interop behavior and reduce our maintenance costs with fewer lines-of-code.

Reply:
I agree with removing the @JvmField annotations. Leveraging Kotlin’s default interop behavior will simplify our code and reduce maintenance by eliminating unnecessary boilerplate. I have checked all relevant directories, and these annotations are not used elsewhere.